### ISSUE TITLE
Basic oauth2 authentication issue

### ISSUE THREAD

**Issue by hobakill** [influxdata/chronograf#4592]
I have religiously followed the instructions here:
https://docs.influxdata.com/chronograf/v1.6/administration/managing-security/#configuring-github-authentication

i'm not seeing any oauth prompts anywhere obvious.

i'm on the latest influxdb and chronograph. i'm using nginx as a proxy server. any help would be appreciated. i don't see anything obvious in the documentation or logs.

**Comment by hobakill** [comment]
FWIW i figured it out but I think the documentation I linked to above is not super clear. I was able to get `oauth2` working if i started the binary from the command line with flags. 

after adding the environment variables to `/etc/default/chronograf` the authorization worked as expected using `systemctl start chronograf`

i may submit a PR for the documentation assuming my logic is sound. setting and `export` variable on its own was not enough. if the `export` needs to be done on the `chronograf` user then that should documented.
